test/functional/uuid/uuid.ts

```diff
@@ -7,10 +7,16 @@ import {createTestingConnections, closeTestingConnections} from "../../utils/tes
 describe("uuid type", () => {
 
     let connections: Connection[];
-    before(async () => connections = await createTestingConnections({
-        entities: [Record],
-        enabledDrivers: ["postgres"] // because only postgres supports jsonb type
-    }));
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Record],
+            enabledDrivers: ["postgres"] // because only postgres supports uuid type
+        });
+
+        await Promise.all(connections.map(async connection => {
+            await connection.entityManager.query(`CREATE extension IF NOT EXISTS "uuid-ossp"`);
```

Comment:
now you even dont need async/await you can just do `connection => connection.entityManager.query(`CREATE extension IF NOT EXISTS "uuid-ossp"`);`